packages/mobile/src/identity/reducer.ts

```diff
@@ -34,10 +34,13 @@ export enum AddressValidationType {
 }
 
 export interface SecureSendPhoneNumberMapping {
-  [e164Number: string]: {
-    address: string | undefined
-    addressValidationType: AddressValidationType
-  }
+  [e164Number: string]: SecureSendDetails
+}
+
+export interface SecureSendDetails {
+  address: string | undefined
+  addressValidationType: AddressValidationType
+  isFetchingAddresses: boolean
```

Comment:
Moving this here seems fine, just curious is there a situation where we're fetching two addresses at once?

Reply:
I was using `isFetchingAddressesSelector` in every payment request item so whenever an action in one item would result in it changing, it would cause a rerender of every payment request item. Realized I needed it to be tied to a specific address so that I could create a more granular selector.